src/sage/stats/distributions/__init__.py

```diff
@@ -0,0 +1,3 @@
+from .discrete_gaussian_integer import DGI
+from .discrete_gaussian_lattice import DGL
+from .discrete_gaussian_polynomial import DGP
```

Comment:
Generally speaking, we keep the `__init__.py` files empty. The `all.py` files handle the imports.